EventMixing/src/Mu2eProductMixer.cc

```diff
@@ -43,11 +43,14 @@ namespace mu2e {
   //----------------------------------------------------------------
   Mu2eProductMixer::Mu2eProductMixer(const Config& conf, art::MixHelper& helper)
     : mixVolumes_(false)
+      , applyTimeOffset_{! conf.simTimeOffset().empty() }
+      , timeOffsetTag_{ conf.simTimeOffset() }
+      , stoff_(0.0)
   {
 
     for(const auto& e: conf.genParticleMixer().mixingMap()) {
       helper.declareMixOp
-        (e.inTag, e.resolvedInstanceName(), &Mu2eProductMixer::mixGenParticles, *this);
+	(e.inTag, e.resolvedInstanceName(), &Mu2eProductMixer::mixGenParticles, *this);
```

Comment:
Is this a random whitespace change?

Reply:
it is the result of automatic whitespace consistency checking by vim, not random.